### ISSUE TITLE
Error when using partition_pdf with parameter infer_table_structure set to true

### ISSUE THREAD

**Issue by iCanDoAllThingszz** [Unstructured-IO/unstructured#3379]
when i invoke the partition_pdf  and set the infer_table_structure as true, the program will exit with error code 139 (interrupted by signal 11: SIGSEGV)。

```python
elements = partition_pdf(path, strategy='hi_res', hi_res_model_name='yolox', infer_table_structure=True, form_extraction_skip_tables=[])

````
**Error message:**Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

**package info：**
# Name                    Version                   Build  Channel
absl-py                   2.0.0                    pypi_0    pypi
aiohttp                   3.8.5           py311h80987f9_0  
aiosignal                 1.2.0              pyhd3eb1b0_0  
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
anyio                     4.0.0                    pypi_0    pypi
appnope                   0.1.3                    pypi_0    pypi
argon2-cffi               23.1.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0                   pypi_0    pypi
arrow                     1.3.0                    pypi_0    pypi
asttokens                 2.4.1                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
async-lru                 2.0.4                    pypi_0    pypi
async-timeout             4.0.2           py311hca03da5_0  
attrs                     23.1.0          py311hca03da5_0  
babel                     2.13.1                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
backoff                   2.2.1                    pypi_0    pypi
beautifulsoup4            4.12.2                   pypi_0    pypi
blas                      1.0                    openblas  
bleach                    6.1.0                    pypi_0    pypi
bottleneck                1.3.5           py311ha0d4635_0  
brotli                    1.0.9                h1a28f6b_7  
brotli-bin                1.0.9                h1a28f6b_7  
brotli-python             1.0.9           py311h313beb8_7  
bzip2                     1.0.8                h620ffc9_4  
ca-certificates           2023.08.22           hca03da5_0  
cachetools                5.3.2                    pypi_0    pypi
certifi                   2023.11.17      py311hca03da5_0  
cffi                      1.16.0                   pypi_0    pypi
chardet                   5.2.0                    pypi_0    pypi
charset-normalizer        3.3.1                    pypi_0    pypi
ci-info                   0.3.0                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
colorama                  0.4.6           py311hca03da5_0  
coloredlogs               15.0.1                   pypi_0    pypi
comm                      0.1.4                    pypi_0    pypi
configobj                 5.0.8                    pypi_0    pypi
configparser              7.0.0                    pypi_0    pypi
contourpy                 1.0.5           py311h48ca7d4_0  
cryptography              41.0.3          py311hd4332d6_0  
cycler                    0.11.0             pyhd3eb1b0_0  
dataclasses-json          0.6.7                    pypi_0    pypi
debugpy                   1.8.0                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
deepdiff                  7.0.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
deprecated                1.2.14                   pypi_0    pypi
effdet                    0.4.1                    pypi_0    pypi
emoji                     2.12.1                   pypi_0    pypi
et_xmlfile                1.1.0           py311hca03da5_0  
etelemetry                0.3.1                    pypi_0    pypi
executing                 2.0.1                    pypi_0    pypi
fastjsonschema            2.18.1                   pypi_0    pypi
filelock                  3.13.1                   pypi_0    pypi
filetype                  1.2.0                    pypi_0    pypi
fitz                      0.0.1.dev2               pypi_0    pypi
flatbuffers               23.5.26                  pypi_0    pypi
fonttools                 4.25.0             pyhd3eb1b0_0  
fqdn                      1.5.1                    pypi_0    pypi
freetype                  2.12.1               h1192e45_0  
frozenlist                1.3.3           py311h80987f9_0  
fsspec                    2023.12.1                pypi_0    pypi
gast                      0.5.4                    pypi_0    pypi
giflib                    5.2.1                h80987f9_3  
google-api-core           2.19.1                   pypi_0    pypi
google-auth               2.25.1                   pypi_0    pypi
google-auth-oauthlib      1.1.0                    pypi_0    pypi
google-cloud-vision       3.7.2                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
googleapis-common-protos  1.63.2                   pypi_0    pypi
grpcio                    1.64.1                   pypi_0    pypi
grpcio-status             1.62.2                   pypi_0    pypi
h11                       0.14.0                   pypi_0    pypi
h5py                      3.10.0                   pypi_0    pypi
httpcore                  1.0.5                    pypi_0    pypi
httplib2                  0.22.0                   pypi_0    pypi
httpx                     0.27.0                   pypi_0    pypi
huggingface-hub           0.23.4                   pypi_0    pypi
humanfriendly             10.0                     pypi_0    pypi
idna                      3.4             py311hca03da5_0  
iopath                    0.1.10                   pypi_0    pypi
ipykernel                 6.26.0                   pypi_0    pypi
ipython                   8.16.1                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                8.1.1                    pypi_0    pypi
isodate                   0.6.1                    pypi_0    pypi
isoduration               20.11.0                  pypi_0    pypi
jedi                      0.19.1                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
joblib                    1.2.0           py311hca03da5_0  
jpeg                      9e                   h80987f9_1  
json5                     0.9.14                   pypi_0    pypi
jsonpath-python           1.0.6                    pypi_0    pypi
jsonpointer               2.4                      pypi_0    pypi
jsonschema                4.19.1                   pypi_0    pypi
jsonschema-specifications 2023.7.1                 pypi_0    pypi
jupyter                   1.0.0                    pypi_0    pypi
jupyter-client            8.5.0                    pypi_0    pypi
jupyter-console           6.6.3                    pypi_0    pypi
jupyter-core              5.4.0                    pypi_0    pypi
jupyter-events            0.8.0                    pypi_0    pypi
jupyter-lsp               2.2.0                    pypi_0    pypi
jupyter-server            2.9.1                    pypi_0    pypi
jupyter-server-terminals  0.4.4                    pypi_0    pypi
jupyterlab                4.0.7                    pypi_0    pypi
jupyterlab-pygments       0.2.2                    pypi_0    pypi
jupyterlab-server         2.25.0                   pypi_0    pypi
jupyterlab-widgets        3.0.9                    pypi_0    pypi
keras                     2.15.0                   pypi_0    pypi
keras-preprocessing       1.1.2              pyhd3eb1b0_0  
kiwisolver                1.4.4           py311h313beb8_0  
langdetect                1.0.9                    pypi_0    pypi
layoutparser              0.3.4                    pypi_0    pypi
lcms2                     2.12                 hba8e193_0  
lerc                      3.0                  hc377ac9_0  
libbrotlicommon           1.0.9                h1a28f6b_7  
libbrotlidec              1.0.9                h1a28f6b_7  
libbrotlienc              1.0.9                h1a28f6b_7  
libclang                  16.0.6                   pypi_0    pypi
libcxx                    14.0.6               h848a8c0_0  
libdeflate                1.17                 h80987f9_1  
libffi                    3.4.4                hca03da5_0  
libgfortran               5.0.0           11_3_0_hca03da5_28  
libgfortran5              11.3.0              h009349e_28  
libopenblas               0.3.21               h269037a_0  
libpng                    1.6.39               h80987f9_0  
libtiff                   4.5.1                h313beb8_0  
libwebp                   1.3.2                ha3663a8_0  
libwebp-base              1.3.2                h80987f9_0  
llvm-openmp               14.0.6               hc6e5704_0  
looseversion              1.3.0                    pypi_0    pypi
lxml                      5.2.2                    pypi_0    pypi
lz4-c                     1.9.4                h313beb8_0  
markdown                  3.5.1                    pypi_0    pypi
markupsafe                2.1.3                    pypi_0    pypi
marshmallow               3.21.3                   pypi_0    pypi
matplotlib-base           3.7.2           py311h7aedaa7_0  
matplotlib-inline         0.1.6                    pypi_0    pypi
mistune                   3.0.2                    pypi_0    pypi
ml-dtypes                 0.2.0                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
multidict                 6.0.2           py311h80987f9_0  
munkres                   1.1.4                      py_0  
mypy-extensions           1.0.0                    pypi_0    pypi
nbclient                  0.8.0                    pypi_0    pypi
nbconvert                 7.9.2                    pypi_0    pypi
nbformat                  5.9.2                    pypi_0    pypi
ncurses                   6.4                  h313beb8_0  
nest-asyncio              1.6.0                    pypi_0    pypi
networkx                  3.2.1                    pypi_0    pypi
nibabel                   5.2.1                    pypi_0    pypi
nipype                    1.8.6                    pypi_0    pypi
nltk                      3.8.1                    pypi_0    pypi
notebook                  7.0.6                    pypi_0    pypi
notebook-shim             0.2.3                    pypi_0    pypi
numexpr                   2.8.7           py311h6dc990b_0  
numpy                     1.26.0          py311he598dae_0  
numpy-base                1.26.0          py311hfbfe69c_0  
oauthlib                  3.2.2                    pypi_0    pypi
omegaconf                 2.3.0                    pypi_0    pypi
onnx                      1.16.1                   pypi_0    pypi
onnxruntime               1.18.1                   pypi_0    pypi
openai                    0.27.4          py311hca03da5_0  
opencv-python             4.10.0.84                pypi_0    pypi
openjpeg                  2.3.0                h7a6adac_2  
openpyxl                  3.0.10          py311h80987f9_0  
openssl                   3.0.12               h1a28f6b_0  
opt-einsum                3.3.0                    pypi_0    pypi
ordered-set               4.1.0                    pypi_0    pypi
overrides                 7.4.0                    pypi_0    pypi
packaging                 23.2                     pypi_0    pypi
pandas                    2.1.1           py311h7aedaa7_0  
pandas-stubs              1.5.3.230203    py311hca03da5_0  
pandocfilters             1.5.0                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pathlib                   1.0.1                    pypi_0    pypi
pdf2image                 1.17.0                   pypi_0    pypi
pdfminer-six              20231228                 pypi_0    pypi
pdfplumber                0.11.1                   pypi_0    pypi
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pikepdf                   9.0.0                    pypi_0    pypi
pillow                    10.0.1          py311h3b245a6_0  
pillow-heif               0.16.0                   pypi_0    pypi
pip                       23.3            py311hca03da5_0  
platformdirs              3.11.0                   pypi_0    pypi
plotly                    5.9.0           py311hca03da5_0  
portalocker               2.10.0                   pypi_0    pypi
prometheus-client         0.17.1                   pypi_0    pypi
prompt-toolkit            3.0.39                   pypi_0    pypi
proto-plus                1.24.0                   pypi_0    pypi
protobuf                  4.23.4                   pypi_0    pypi
prov                      2.0.1                    pypi_0    pypi
psutil                    5.9.6                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
py-cpuinfo                9.0.0                    pypi_0    pypi
pyasn1                    0.5.1                    pypi_0    pypi
pyasn1-modules            0.3.0                    pypi_0    pypi
pycocotools               2.0.8                    pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0  
pydot                     2.0.0                    pypi_0    pypi
pygments                  2.16.1                   pypi_0    pypi
pymupdf                   1.24.7                   pypi_0    pypi
pymupdfb                  1.24.6                   pypi_0    pypi
pyopenssl                 23.2.0          py311hca03da5_0  
pyparsing                 3.0.9           py311hca03da5_0  
pypdf                     4.2.0                    pypi_0    pypi
pypdfium2                 4.30.0                   pypi_0    pypi
pysocks                   1.7.1           py311hca03da5_0  
pytesseract               0.3.10                   pypi_0    pypi
python                    3.11.5               hb885b13_0  
python-dateutil           2.8.2              pyhd3eb1b0_0  
python-iso639             2024.4.27                pypi_0    pypi
python-json-logger        2.0.7                    pypi_0    pypi
python-magic              0.4.27                   pypi_0    pypi
python-multipart          0.0.9                    pypi_0    pypi
python-tzdata             2023.3             pyhd3eb1b0_0  
pytz                      2023.3.post1    py311hca03da5_0  
pyxnat                    1.6.2                    pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
pyzmq                     25.1.1                   pypi_0    pypi
qtconsole                 5.4.4                    pypi_0    pypi
qtpy                      2.4.1                    pypi_0    pypi
rapidfuzz                 3.9.3                    pypi_0    pypi
rdflib                    6.3.2                    pypi_0    pypi
readline                  8.2                  h1a28f6b_0  
referencing               0.30.2                   pypi_0    pypi
regex                     2023.10.3                pypi_0    pypi
reportlab                 4.2.2                    pypi_0    pypi
requests                  2.31.0          py311hca03da5_0  
requests-oauthlib         1.3.1                    pypi_0    pypi
requests-toolbelt         1.0.0                    pypi_0    pypi
rfc3339-validator         0.1.4                    pypi_0    pypi
rfc3986-validator         0.1.1                    pypi_0    pypi
rpds-py                   0.10.6                   pypi_0    pypi
rsa                       4.9                      pypi_0    pypi
safetensors               0.4.3                    pypi_0    pypi
scikit-learn              1.2.2           py311h313beb8_1  
scipy                     1.11.3          py311hc76d9b0_0  
seaborn                   0.13.2                   pypi_0    pypi
send2trash                1.8.2                    pypi_0    pypi
sentencepiece             0.2.0                    pypi_0    pypi
setuptools                68.0.0          py311hca03da5_0  
simplejson                3.19.2                   pypi_0    pypi
six                       1.16.0             pyhd3eb1b0_1  
sniffio                   1.3.0                    pypi_0    pypi
soupsieve                 2.5                      pypi_0    pypi
sqlite                    3.41.2               h80987f9_0  
stack-data                0.6.3                    pypi_0    pypi
sympy                     1.12                     pypi_0    pypi
tabulate                  0.9.0                    pypi_0    pypi
tenacity                  8.2.2           py311hca03da5_0  
tensorboard               2.15.1                   pypi_0    pypi
tensorboard-data-server   0.7.2                    pypi_0    pypi
tensorflow                2.15.0                   pypi_0    pypi
tensorflow-estimator      2.15.0                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.34.0                   pypi_0    pypi
tensorflow-macos          2.15.0                   pypi_0    pypi
termcolor                 2.4.0                    pypi_0    pypi
terminado                 0.17.1                   pypi_0    pypi
tesseract                 0.1.3                    pypi_0    pypi
threadpoolctl             2.2.0              pyh0d69192_0  
tiktoken                  0.5.1                    pypi_0    pypi
timm                      1.0.7                    pypi_0    pypi
tinycss2                  1.2.1                    pypi_0    pypi
tk                        8.6.12               hb8d0fd4_0  
tokenizers                0.19.1                   pypi_0    pypi
torch                     2.3.1                    pypi_0    pypi
torchvision               0.18.1                   pypi_0    pypi
tornado                   6.3.3                    pypi_0    pypi
tqdm                      4.65.0          py311hb6e6a13_0  
traitlets                 5.12.0                   pypi_0    pypi
traits                    6.3.2                    pypi_0    pypi
transformers              4.42.3                   pypi_0    pypi
types-python-dateutil     2.8.19.14                pypi_0    pypi
types-pytz                2022.4.0.0      py311hca03da5_1  
typing-extensions         4.8.0                    pypi_0    pypi
typing-inspect            0.9.0                    pypi_0    pypi
tzdata                    2023c                h04d1e81_0  
ultralytics               8.2.48                   pypi_0    pypi
ultralytics-thop          2.0.0                    pypi_0    pypi
unstructured              0.14.9                   pypi_0    pypi
unstructured-client       0.23.8                   pypi_0    pypi
unstructured-inference    0.7.36                   pypi_0    pypi
unstructured-pytesseract  0.3.12                   pypi_0    pypi
uri-template              1.3.0                    pypi_0    pypi
urllib3                   2.0.7                    pypi_0    pypi
utils                     1.0.2                    pypi_0    pypi
wcwidth                   0.2.8                    pypi_0    pypi
webcolors                 1.13                     pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.6.4                    pypi_0    pypi
werkzeug                  3.0.1                    pypi_0    pypi
wheel                     0.41.2          py311hca03da5_0  
widgetsnbextension        4.0.9                    pypi_0    pypi
wrapt                     1.14.1                   pypi_0    pypi
xz                        5.4.2                h80987f9_0  
yarl                      1.8.1           py311h80987f9_0  
zlib                      1.2.13               h5a0b063_0  
zstd                      1.5.5                hd90d995_0
